RadixWallet/Core/SharedModels/P2P/Dapp/Models/InteractionResponse/WalletInteractionSuccessResponseItems.swift

```diff
@@ -3,13 +3,16 @@ extension P2P.Dapp.Response.WalletInteractionSuccessResponse {
 	public enum Items: Sendable, Hashable, Encodable {
 		case request(RequestResponseItems)
 		case transaction(TransactionResponseItems)
+		case verify
```

Comment:
This was added to be returned in [DappInteractionModels](https://github.com/radixdlt/babylon-wallet-ios/pull/1173/files#diff-0e718d69a44a59e23c1d7b96890d7c3ef235cacb310a02a2381346c06ccb0ab2R286-R291). However, such `init` isn't ever called for the `.verify` case, which makes me wonder if I am missing something on `DappInteractionFlow`.